File: AppSistemaVenta/src/app/Interfaces/auditoria.ts
export interface Auditoria {
    auditoriaId: number;
    idUsuario?: number | null;
    usuario?: string | null;
    accion?: string | null;
    entidadAfectada?: string | null;
    fechaAccion?: Date | null;
    detalleAccion?: string | null;
  }

File: APISistemaVenta/SistemaVenta.Model/Auditoria.cs
using System;
using System.Collections.Generic;

namespace SistemaVenta.Model;

public partial class Auditoria
{
    public int AuditoriaId { get; set; }

    public int? IdUsuario { get; set; }

    public string? Usuario { get; set; }

    public string? Accion { get; set; }

    public string? EntidadAfectada { get; set; }

    public DateTime? FechaAccion { get; set; }

    public string? DetalleAccion { get; set; }
}
